int32 traj_id
---
snaak_manipulation/msg/RigidTransform end_pose
---
# TODO: want any feedback?